openadapt/record.py

```diff
@@ -642,5 +642,10 @@ def record(
     logger.info(f"saved {recording_timestamp=}")
 
 
+# entry point
+def start():
```

Comment:
What is this function for?

Reply:
we need a function that acts an entry point / main() for us to use `poetry run`, called like `openadapt.record:start`. Please let me know if there's a better way to do this, I am aware there's a guard for this already but openadapt.record cannot be called through poetry this way.